packages/augur-ui/src/assets/styles/_colors.less

```diff
@@ -185,6 +185,7 @@
   --color-open-40: fade(#18C88D, 40%);
   --color-reporting: #F59300;
   --color-reporting-15: fade(#F59300, 15%);
+  --color-reporting-30: fade(#F59300, 30%);
```

Comment:
gotta add a version for the root as well, if it's undefined in trading it may cause issues. trading also uses a slightly different color for reporting so we should make sure it has a 30% fade as well for completeness sake.